src/backend/cdb/motion/ic_udpifc.c

```diff
@@ -1167,6 +1169,107 @@ resetRxThreadError()
 }
 
 
+/*
+ * isIPv6Available
+ * 		Check whether a valid IPv6 address exists in the system.
+ *
+ * 		We want to have both loopback IPv6 address, which will show if IPv6 was
```

Comment:
This statement sounds a bit awkward to me. I suggest to rephrase it to ensure it is not misinterpreted.
For ex.:
```
* To ensure the IPv6 is available in the system, we must check two conditions:
* 1. there is at least one non-link-local IPv6 address available for our use;
* 2. and the loopback IPv6 address exists (as address from item#1 could 
* appear if the interface is reset after disabling of IPv6 in the host system).
```